antlr-kotlin-gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+@file:Suppress("UnstableApiUsage")
```

Comment:
I assume the code of the gradle plugin has been restored as-is, and so it does not need to be reviewed at this stage

Reply:
Yes, it's not edited since I last removed it, so no need to review it.

Reply:
Good, closing the conversation